### DISCUSSION TITLE
ViewScene Multiple Windows

### DISCUSSION THREAD

**Discussion by salmanfarsi100** [SCIInstitute/SCIRun#2426] [Q&A]
Hi! Does anyone know how to open multiple windows of the same ViewScene? We had this option in SCIRun4, but I don't see it in SCIRun5.

**Comment by jessdtate** [comment]
Instead of using multiple windows on the same viewscene module, SCIRun 5 allows users to lock camera movements using the group function (in the advanced menu, the icon looks like a chain link).
